docs/finality-provider-phase2.md

```diff
@@ -0,0 +1,589 @@
+{"addr": "127.0.0.1:2113"} 
+2024-10-30T12:42:29.393278Z     info    RPC server listening    {"address": "127.0.0.1:12582"} 
+2024-10-30T12:42:29.393363Z     info    EOTS Manager Daemon is fully active!  
+EOTS Manager Daemon is fully active!  
+```
+
+>**Note**: It is recommended to run the `eotsd` daemon on a separate machine or
+network segment to enhance security. This helps isolate the key management
+functionality and reduces the potential attack surface. You can edit
+the`EOTSManagerAddress` in the configuration file of the finality provider to
+reference the address of the machine where `eotsd` is running.
+
+## Setting up the Finality Provider
+
+The Finality Provider Daemon (FPD) is responsible for monitoring new Babylon blocks, 
+committing public randomness for the blocks it intends to provide finality signatures for, 
+and submitting finality signatures. For more information on Finality Providers, please see 
+[here](#).
+
+### Step 1: Initialize the Finality Provider Daemon
+
+Use the fpd init command to initialize a home directory for the Finality Provider. 
+You can set or change the home directory using the `--home` tag. For example, use 
+`--home ./fpKeys` to specify a custom directory.
+
+```shell 
+fpd init --home <path> 
+```
+
+>Note: Running this command may return the message 
+`service injective.evm.v1beta1.Msg does not have cosmos.msg.v1.service proto annotation`, 
+which is expected and can be ignored.
+
+### Step 2: Add key for the Finality Provider on the Babylon Chain
+
+The keyring is maintained by the finality provider daemon, this is local storage
+of the keys that the daemon uses. The account associated with this key exists on
+the babylon chain.
+
+Use the following command to add a key for your finality provider:
+
+```shell 
+fpd keys add --keyname <key-name> --keyring-backend test --home <path>
+```
+
+We use `--keyring-backend test`, which specifies which backend to use for the
+keyring, `test` stores keys unencrypted on disk. For production environments,
+use `file` or `os` backend.
+
+
+ There are three options for the keyring backend:
+
+ `test`: Stores keys unencrypted on disk. It’s meant for testing purposes and
+ should never be used in production.  
+ `file`: Stores encrypted keys on disk,
+ which is a more secure option than test but less secure than using the OS
+ keyring.  
+ `os`: Uses the operating system's native keyring, providing the
+ highest level of security by relying on OS-managed encryption and access
+ controls.
+
+This command will create a new key pair and store it in your keyring. The output
+should look similar to the below.
+
+``` shell
+- address: bbn19gulf0a4yz87twpjl8cxnerc2wr2xqm9fsygn9
+  name: finality-provider pubkey:
+  '{"@type":"/cosmos.crypto.secp256k1.PubKey",
+  "key":"AhZAL00gKplLQKpLMiXPBqaKCoiessoewOaEATKd4Rcy"}'
+  type: local
+```
+
+>Note: Please verify the `chain-id` from the Babylon RPC
+node [https://rpc.testnet5.babylonlabs.io/status]
```

Comment:
Instead of pointing to an RPC node here, I think it's better that we link to the networks page dedicated for the testnet. This page will include RPCs, seeds, and the chain ID

Reply:
I think more general. We will have a README with basic information about the chain (e.g. chain-id), RPC and seed node providers, genesis file, etc.